src/client/common/experiments/service.ts

```diff
@@ -115,7 +115,8 @@ export class ExperimentService implements IExperimentService {
         // https://github.com/microsoft/vscode-jupyter/issues/4771
         if (
             experiment === ExperimentGroups.NativeNotebook &&
-            this.extensions.getExtension('ms-dotnettools.dotnet-interactive-vscode')
+            (this.extensions.getExtension('ms-dotnettools.dotnet-interactive-vscode') ||
```

Comment:
This looks like it could look at distinct values from extensionsThatSupportJupyterKernelLanguages possibly if we just wanted to add the extension names in one place. Or maybe just have the two extension names as constants.

Reply:
I'm not too concerned about this, as we'll be going 100% with native editor soon